scripts/woltka_merge

```diff
@@ -0,0 +1,97 @@
+#!/usr/bin/env python
+
+import os
+import biom
+import glob as glob_
+import pandas as pd
+import h5py
+import click
+from functools import reduce
+from operator import or_
+
+
+def fast_merge(tables):
```

Comment:
Unless I'm missing something, this functionality is available in the most recent release of BIOM, can we just use that instead?